inst/doc/Processing_Controlled_Data_With_Bioconductor.md

```diff
@@ -52,102 +42,58 @@ bioconductor microarray container (docker pull bioconductor/release_microarray).
 
 ```
 docker run -ti --privileged bioconductor/release_microarray /bin/bash
```

Comment:
I think it would help if:
1) all `docker run` examples included using `-v` to mount a directory outside the container
2) then once inside the container, set the current working directory to be the mounted directory

That way people are less likely to loose work if they exit the docker container without copying images out first.